voorhoede/tudelft-spacefinder
# Feat/conditionally hide opening hours

# Changes

- The opening hours, open/closed status and a corresponding filter are hidden is a flag is set in the .evn file

# Associated issue

Resolves https://trello.com/c/W6sHWXc3/30-remove-opening-hours-as-theyre-outdated-incorrect

# How to test

1. Toggle between 1 and 0 in the .env file for HIDE_OPENING_HOURS. See the opening hours, open/closed status and "show open locations only" filter disappear and appear back in the UI.
2. Make sure that if the mentioned filter was set (and persisted in local storage) and then the setting has been set to "1", the filter is reset upon next page refresh.

# Checklist
